### DISCUSSION TITLE
gm

### DISCUSSION THREAD

**Discussion by tmm** [rainbow-me/rainbowkit#312] [General]
gm

**Comment by markdalgleish** [comment]
gm

**Comment by peterferguson** [comment]
gm

**Comment by nickbytes** [comment]
gm

**Comment by salieflewis** [comment]
gm

**Comment by therealharpaljadeja** [comment]
gm

**Comment by dawksh** [comment]
gm

**Comment by pugson** [comment]
gm

**Comment by Diogomartf** [comment]
gm

**Comment by peduarte** [comment]
gm

**Comment by lochie** [comment]
gm

**Comment by KABILESHTHEEMPEROR** [comment]
npm init @rainbow-me/rainbowkit@latest i have installed this package. and how do i proceed next? there are only less guidance for the rainbowkit integration. someone help please

**Reply by Adebesin-Cell** [reply]
> npm init @rainbow-me/rainbowkit@latest i have installed this package. and how do i proceed next? there are only less guidance for the rainbowkit integration. someone help please

After running the command `npm init @rainbow-me/rainbowkit@latest`, you will be prompted to provide a name for your new app. Once you've entered a name, the setup process will automatically create a new folder with that name and install all the necessary dependencies for your project.

### Next Steps:

1. **Navigate to Your Project Directory**:
   ```bash
   cd <your-app-name>
   ```
   Replace `<your-app-name>` with the name you provided during the setup.

2. **Open Your Project in Visual Studio Code (Optional)**:
   If you're using Visual Studio Code, you can open your project by running:
   ```bash
   code .
   ```

3. **Start the Development Server**:
   To start the development server and see your app in action, run the following command:
   ```bash
   npm run dev
   ```

4. **Begin Customizing Your App**:
   With the development server running, you can begin customizing and building your app. The server will automatically reload your changes as you make updates.

Happy coding! 🚀
